element-connector/src/main/java/org/eclipse/californium/elements/DtlsEndpointContext.java

```diff
@@ -47,7 +47,28 @@
 	 * The name of the attribute that contains the timestamp of the last
 	 * handshake of the DTLS session.
 	 */
-	public static final String KEY_HANDSHAKE_TIMESTAMP = "KEY_HANDSHAKE_TIMESTAMP";
+	public static final String KEY_HANDSHAKE_TIMESTAMP = "DTLS_HANDSHAKE_TIMESTAMP";
+	/**
+	 * The name of the attribute that contains a handshake mode. Values see
+	 * {@link #HANDSHAKE_MODE_FORCE}, {@link #HANDSHAKE_MODE_START}, and
+	 * {@link #HANDSHAKE_MODE_NONE}. None critical attribute, not considered for
+	 * matching.
+	 */
+	public static final String KEY_HANDSHAKE_MODE = KEY_PREFIX_NONE_CRITICAL + "DTLS_HANDSHAKE_MODE";
+
+	/**
+	 * Force handshake before send this message.
+	 */
+	public static final String HANDSHAKE_MODE_FORCE = "force";
```

Comment:
Is it the same as KEY_RESUMPTION_TIMEOUT with 0 value (but more intuitive?)

Reply:
Yes, therefore I removed the javadoc for the 0 stuff :-).

Reply:
Missing removing javadoc few line below in this class (l.52)